### DISCUSSION TITLE
Change Codec to Nvidia NVENC

### DISCUSSION THREAD

**Discussion by invictusro** [scivision/PyLivestream#90] [Q&A]
Hello, does any of you have any ideea how can I change the codec from the x264 that is now to the NVIDIA Nvenc ?

**Comment by scivision** [comment]
https://github.com/scivision/PyLivestream/commit/41e7ec4c3911b966f877f7c1b5113314e2eec10b adds "video_codec" parameter in the JSON file to allow for this. I haven't tested with that codec.
